packages/@react-spectrum/actionbar/src/ActionBar.tsx

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {ActionButton} from '@react-spectrum/button';
+import {ActionGroup} from '@react-spectrum/actiongroup';
+import {classNames, useDOMRef, useStyleProps} from '@react-spectrum/utils';
+import CrossLarge from '@spectrum-icons/ui/CrossLarge';
+import {DOMRef} from '@react-types/shared';
+import {filterDOMProps} from '@react-aria/utils';
+// @ts-ignore
+import intlMessages from '../intl/*.json';
+import {Provider, useProviderProps} from '@react-spectrum/provider';
+import React from 'react';
+import {SpectrumActionBarProps} from '@react-types/actionbar';
+import styles from './actionbar.css';
+import {Text} from '@react-spectrum/text';
+import {useMessageFormatter} from '@react-aria/i18n';
+
+function ActionBar(props: SpectrumActionBarProps, ref: DOMRef<HTMLDivElement>) {
+  // Grabs specific props from the closest Provider (see https://react-spectrum.adobe.com/react-spectrum/Provider.html#property-groups). Remove if your component doesn't support any of the listed props.
+  props = useProviderProps(props);
+
+  const {
+    children,
+    isEmphasized,
+    onAction,
+    onClearSelection,
+    selectedItemCount,
+    variant,
+    ...otherProps
+  } = props;
+
+  const innerDivRef = React.useRef<HTMLDivElement>();
+  const height = innerDivRef.current?.clientHeight;
```

Comment:
this mostly works, but it still causes the entire ActionBarContainer to change size, the table should shrink by the amount that the ActionBar grows
also, best to use a `useLayoutEffect` and not do it directly in the render

this is more of a note to the rest of the team to get some opinions:
I know it should probably always be the same height, but on the off change there's a really long string and small width table, we may also need to listen to size changes, should we use https://github.com/adobe/react-spectrum/blob/main/packages/%40react-aria/utils/src/useResizeObserver.ts or should this be impossible?